src/Propel/Runtime/Map/DatabaseMap.php

```diff
@@ -38,14 +38,14 @@ class DatabaseMap
     /**
      * Tables in the database, using table name as key
      *
-     * @var array<\Propel\Runtime\Map\TableMap>
+     * @var array<\Propel\Runtime\Map\TableMap|class-string>
      */
     protected $tables = [];
 
     /**
      * Tables in the database, using table phpName as key
      *
-     * @var array<\Propel\Runtime\Map\TableMap>
+     * @var array<\Propel\Runtime\Map\TableMap|class-string>
```

Comment:
Should this be `...|class-string<\Propel\Runtime\Map\TableMap>>` ?

Reply:
Ah, I didn't know this works. Nice!

Reply:
Well, `PHPDoc tag @var has invalid value` 
I guess we also need to adjust the sniffer then to allow this kind of syntax :)

Reply:
Or is PHPStan not handling this well?
Because the syntax should be working as such

Reply:
but it seems to work, doesn't it?